sickbeard/providers/newznab.py

```diff
@@ -195,11 +200,11 @@ def _get_episode_search_strings(self, ep_obj, add_string=''):
                 params['q'] = helpers.sanitizeSceneName(cur_exception)
                 paramsNoEp = params.copy()
                 
-                paramsNoEp['q'] = paramsNoEp['q'] + " " + str(paramsNoEp['ep'])
-                if "ep" in paramsNoEp:
-                    paramsNoEp.pop("ep")
+                paramsNoEp['q'] = paramsNoEp['q'] + " " + str(paramsNoEp[episode])
```

Comment:
There should be no ep # in the q string if its added as ep/episode parameter. Thats why there is params and paramsNoEp